src/DynamoDbQueryBuilder.php

```diff
@@ -154,13 +155,34 @@ public function find($id, array $columns = [])
         return $model;
     }
 
+    public function findOrFail($id, array $columns = [])
+    {
+        $result = $this->find($id, $columns);
+        
+        if (is_array($id) && count($result) == count(array_unique($id))) {
+            return $result;
+        } elseif (! is_null($result)) {
```

Comment:
Yeah, I modified it to make Nitpick happy ... I just pushed an updated version that is a verbatim "copy/paste" of Eloquent (which was what I had originally for my local tests, until Nitpick got nitpicky ...)